src/main/java/com/capitalone/dashboard/collector/DefaultWhiteSourceClient.java

```diff
@@ -0,0 +1,316 @@
+
+        } catch (Exception e) {
+            LOG.info("Exception occurred while calling getProjectInventory " + e.getMessage());
+        }
+        return libraryPolicyResult;
+    }
+
+
+    @Override
+    public LibraryPolicyResult getProjectAlerts(String instanceUrl, WhiteSourceComponent whiteSourceComponent) {
+        String url = getApiBaseUrl(instanceUrl);
+        LibraryPolicyResult libraryPolicyResult = new LibraryPolicyResult();
+        try {
+            JSONObject jsonObject = makeRestCall(getApiBaseUrl(instanceUrl), Constants.RequestType.getProjectAlerts, null, null, whiteSourceComponent.getProjectToken());
+            JSONArray alerts = (JSONArray) Objects.requireNonNull(jsonObject).get(Constants.ALERTS);
+            if (CollectionUtils.isEmpty(alerts)) {
+                return null;
+            } else {
+                libraryPolicyResult.setCollectorItemId(whiteSourceComponent.getId());
+                libraryPolicyResult.setTimestamp(System.currentTimeMillis());
+                for (Object a : alerts) {
+                    JSONObject alert = (JSONObject) a;
+                    String alertType = getStringValue(alert, Constants.TYPE);
+                    String alertLevel = getStringValue(alert, Constants.LEVEL);
+                    JSONObject library = (JSONObject) Objects.requireNonNull(alert).get(Constants.LIBRARY);
+                    String creationDate = getStringValue(alert, Constants.CREATION_DATE);
+                    String description = getStringValue(alert, Constants.DESCRIPTION);
+                    String componentName = getStringValue(library, Constants.FILENAME);
+                    libraryPolicyResult.setEvaluationTimestamp(getLongValue(alert, Constants.TIME));
+                    // add threat for license
+                    JSONArray licenses = (JSONArray) Objects.requireNonNull(library).get(Constants.LICENSES);
+                    if (!CollectionUtils.isEmpty(licenses)) {
+                        setAllLibraryLicensesAlerts(licenses, libraryPolicyResult, componentName, getDays(creationDate) + "", getLicenseThreatLevel(alertType, alertLevel, description));
+                    }
+                    // add threat for Security vulns
+                    JSONObject vulns = (JSONObject) Objects.requireNonNull(alert).get(Constants.VULNERABILITY);
+                    if (!CollectionUtils.isEmpty(vulns)) {
+                        setSecurityVulns(vulns, libraryPolicyResult, componentName, getDays(creationDate) + "");
+                    }
+
+                }
+            }
+        } catch (Exception e) {
+            LOG.info("Exception occurred while calling getProjectAlerts " + e.getMessage());
+        }
+        return libraryPolicyResult;
+    }
+
+    private JSONObject makeRestCall(String url, Constants.RequestType requestType, String orgToken, String productToken, String projectToken) {
+        LOG.info("collecting analysis for ===> " + url + " and requestType : " + requestType);
```

Comment:
This only provides one more piece of of information, requestType, than the common logging inside restClient.makeRestCallPost. Do not think this will help much for trouble-shooting. If you do need requestType, why not also log org name, but no need to log url. Use name=value format to be Splunk friendly

Reply:
Added logging for orgName and requestType. Removed url